custom_components/tuya_local/devices/warme_gsh_heater.yaml

```diff
@@ -0,0 +1,38 @@
+name: Warme GSH heater
```

Comment:
Top level name should be without branding, as the device may be sold in different markets under different brands.  Just "Heater" is enough - Home Assistant will take care of making it unique if there are multiple (and the end user can edit the device name themselves to make it more specific to their installation).

Reply:
I'll rename

Do you want the yaml file renamed as well? It has 'Warme' in the filename?

Reply:
The file name is OK, the requirement there is that the filename has to be unique, and adding the brand helps with that without resorting to ..._1, ..._2 etc.  The name does not have to be unique as most of the other configs will not be loaded if you don't have a matching device  configured, and even if you do, HA handles adding a number to make them unique.

Reply:
Renamed